.devcontainer/devcontainer.json

```diff
@@ -20,9 +20,10 @@
                 "ms-vscode.cpptools-themes",
                 "ms-vscode.makefile-tools",
                 "Tsinghua-Hexin-Joint-Institute.gem5-slicc",
-                "ms-python.black-formatter",
```

Comment:
How is this not used? Shouldn't we be all using this to format our python code?

Does some other extension supersede this?

Reply:
@BobbyRBruce I am cleaning out these PRs. Can you answer this question?

Reply:
this is a vscode extension, not the the python black utility which is used in pre-commit. The pre-commit python black is installed via pre-commit.

Reply:
This is not all that important. This is simply an unnecessary extension that isn't worth installing. To my knowledge it does no harm.

Reply:
I guess I use it in vscode when I "format document" before committing. I like to see the formatted document before pre-commit makes its changes.

Reply:
I never use it personally but if you do that's fine. I'll quickly update the PR.

Reply:
Done.